### PR TITLE
fix: SwiftLint warnings for rule `multiple_closures_with_trailing_closure`

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

In this PR I remove multiple_closures_with_trailing_closure rule from the disabled rules list and fix all warnings related to it. 

- **Multiple Closures with Trailing Closure Violation**: Trailing closure syntax should not be used when passing more than one closure argument. `(multiple_closures_with_trailing_closure)`

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by billypchan** [comment]
I would prefer disable this rule since it would save some typing and a close bracket at the end is not clear to read.

**Comment by agisilaos** [comment]
@billypchan I will have to disagree with this. Personally, I found it much more clear regarding to what's happening with the code.

File: Wire-iOS/Sources/UserInterface/ConnectRequests/ConnectRequestsViewController.swift [hunk 1]
```diff
@@ -85,8 +85,8 @@ final class ConnectRequestsViewController: UIViewController, UITableViewDataSour
     override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         coordinator.animate(alongsideTransition: { _ in
             self.tableView.reloadData()
-        }) { _ in
-        }
+        }, completion: { _ in
+        })
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/ConnectRequests/ConnectRequestsViewController.swift:89]: I think we can set completion as nil, what do you think?

Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/ConnectRequests/ConnectRequestsViewController.swift:89]: I think so too. Good catch.

**Comment by KaterinaWire** [comment]
> @billypchan I will have to disagree with this. Personally, I found it much more clear regarding to what's happening with the code.

@agisilaos @billypchan  I don’t have a strong opinion, but if we have disagreements, I think it makes sense to discuss with the whole team and get more opinions.

**Review by johnxnguyen — APPROVED** [review]

**Review by billypchan — APPROVED** [review]

**Pull request merged**